core/src/main/java/com/amplifyframework/devmenu/DevMenuEnvironmentFragment.java

```diff
@@ -38,6 +44,27 @@ public DevMenuEnvironmentFragment() { }
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.dev_menu_fragment_environment, container, false);
+        View envInfoView = inflater.inflate(R.layout.dev_menu_fragment_environment, container, false);
+        displayEnvironmentInfo((TextView) envInfoView.findViewById(R.id.env_info_text));
+        return envInfoView;
+    }
+
+    /**
+     * Displays the environment information on the given TextView.
+     * @param envInfoTextView the TextView that will contain the environment information text
+     */
+    private void displayEnvironmentInfo(TextView envInfoTextView) {
+        SpannableStringBuilder stringBuilder = new SpannableStringBuilder();
+        SpannableString pluginInfoTitle = new SpannableString("Amplify Plugins Information");
+        pluginInfoTitle.setSpan(new StyleSpan(Typeface.BOLD), 0, pluginInfoTitle.length(),
+                Spannable.SPAN_EXCLUSIVE_EXCLUSIVE);
```

Comment:
Even though they would be _short_ helper methods -- there is an opporrunity here to create some semantically-named ones. You can probably get this line to like `setBold(pluginIntoTitle)` or something? Same general approach applies to each line in this method where your intention is to `setVisualStylingOfSomeKind(...)`.